package.json

```diff
@@ -29,15 +28,14 @@
     "kang": "0.0.8",
     "less": "1.7.5",
     "mime": "1.2.11",
-    "moray": "git+ssh://git@github.com:joyent/node-moray.git#b84ef0e",
+    "moray": "git+ssh://git@github.com:joyent/node-moray.git#31c0902",
     "node-uuid": "1.4.3",
     "promise": "6.1.0",
     "react": "0.13.0",
     "reactify": "1.0.0",
     "restify": "2.8.5",
-    "sdc-clients": "9.2.0",
-    "ufds": "1.1.3",
-    "sdc-events": "git+ssh://git@github.com:joyent/sdc-events.git",
+    "sdc-clients": "9.5.0",
+    "ufds": "git+ssh://git@github.com:joyent/node-ufds.git#98a563d",
```

Comment:
Please do not change from an NPM dependency to a Git URL for `ufds`.  If we need to publish a new version of this module, we should do that.